src/simulation/dynamics/facetSRPDynamicEffector/facetSRPDynamicEffector.h

```diff
@@ -54,15 +56,21 @@ class FacetSRPDynamicEffector: public SysModel, public DynamicEffector {
                   Eigen::Vector3d normal_B,
                   Eigen::Vector3d locationPntB_B,
                   Eigen::Vector3d rotAxis_B);                                            //!< Method for adding facets to the spacecraft geometry structure
+    void addArticulatedFacet(Message<HingedRigidBodyMsgPayload> *tmpMsg);
```

Comment:
This only works with HingedRigidBody Msgs?  Each panel of spinning bodies, double-hinged, etc. effectors puts out an inertial state.  In the future we could use these states.  For now I"m ok with the HingedRigidBody msg to start with.